rollup.config.js

```diff
@@ -13,6 +13,9 @@ import {terser} from 'rollup-plugin-terser';
 // A Rollup plugin for copying files.
 import copy from 'rollup-plugin-copy';
 
+// A Rollup plugin for copying files.
+import json from '@rollup/plugin-json';
```

Comment:
What is this being used for? (Also, looks like the comment might have just been copied from above?)

Reply:
It's used to read the dimension numbers from [the JSON config file](https://github.com/GoogleChrome/developer.chrome.com/blob/5242c85273759dc21535e8d5d99e479617b37807/site/_data/analytics.json) rather than hardcoding them.

https://github.com/GoogleChrome/developer.chrome.com/blob/5242c85273759dc21535e8d5d99e479617b37807/site/_js/analytics.js#L49-L50

Will update the comment.

Reply:
Oh, I see; it's to make this line work:

```js
import {dimensions} from '../_data/analytics.json';
```

Reply:
Aside: not worth doing for just this PR, but I'm curious if there's a more standard-compliant way to do this with Rollup, e.g. https://bobbyhadz.com/blog/javascript-import-json-file

Reply:
Actually when I tried to import this rollup TOLD me to install this plugin! 😀

I checked and web.dev uses something different but was beyond me, and this was a simple suggested plugin.